08_project/willshen8/cmd/puppy-server/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	puppy "github.com/anz-bank/go-course/08_project/willshen8/pkg/puppy"
```

Comment:
You don't need to alias these packages. The imported name is always the last path component.

So `"github.com/anz-bank/go-course/08_project/willshen8/pkg/puppy"` will import as `puppy`